### DISCUSSION TITLE
neper -V with real values

### DISCUSSION THREAD

**Discussion by kshankar7** [neperfepx/neper#833] [Q&A] [answered]
Hi,
I want to plot a mesh with coloring each grain with real values from a separate file. I am using the following command but unable to produce the intended plot. The colors tend to gravitate only towards the extreme, even though there are values in the middle. For this, I am using Neper 4.6.1. Can someone help me to check if I am doing this correctly? @rquey 

```
neper -V simulation.msh \
	-dataelset3dcol "real:file(confidence.txt)" \
	-dataelset3dscale 0.5:1.0 \
	-dataelt3dedgerad 0 \
	-cameracoo 5:5:5 \
	-camerasky 0:1:0 \
	-print confidence
```

**Comment by rquey** [comment, marked as answer]
The command seems OK. The color scale should be as specified by `-dataelset3dscale 0.5:1.0`. You may need to adjust it...

**Comment by kshankar7** [comment]
Thank you @rquey
